### ISSUE TITLE
[BUG] SKImage.FromEncodedData(String) returns image with broken alpha on GPU backend

### ISSUE THREAD

**Issue by antongit** [mono/SkiaSharp#1052]
**Description**

We are using 1.68.1 and see that SKImage.FromEncodedData(String) loads PNG images with broken alpha when drawn on GPU backend.

**Code**

```
using (var image = SKImage.FromEncodedData(path))
{
   canvas.DrawImage(image, new SKPoint(400.0f, 100.0f));
}
```

**Expected & Actual Behavior**

![image](https://user-images.githubusercontent.com/1189037/70338537-6b4d5500-184d-11ea-9f73-7d9c2b2d07de.png)

**Basic Information**

- Version with issue:  1.68.0, 1.68.1
- IDE:  Visual Studio
- Platform Target Frameworks: 
  - Windows Classic:  Windows10
- Target Devices:   
  - Desktop

**Reproduction Link**

[SkiaApp.zip](https://github.com/mono/SkiaSharp/files/3933088/SkiaApp.zip)

**Comment by Mikolaytis** [comment]
This is not the only overload with this behavior. I've faced the same issue with Stream overload if I am remember it correctly. Fixed it with use of another overload.

**Comment by mattleibow** [comment]
I'll have a look and see what I can find out. Maybe something is broken in the decoder that is not using the correct alpha from the file - but - it works in raster...

**Comment by mattleibow** [comment]
Thanks for the issue. I got a fix building and it will be in the 1.68.1.1 release.
